src/css/style.css

```diff
@@ -0,0 +1,274 @@
+/* -------------------Common styles--------------------------- */
+
+* {
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+main,
+footer {
+  background-color: white;
+}
+
+body {
+  background-color: lightgrey !important;
+}
+
+/* ------------------------Header------------ */
+
+.nav1,
+.sign-up,
+.tagger,
+.option-subscribe {
+  background-color: #ff0500 !important;
+}
+
+.nav2,
+.in-the-magazine,
+.user-option {
+  background-color: white !important;
```

Comment:
Instead of using !important please use the available bootstrap colors where possible: https://getbootstrap.com/docs/4.0/utilities/colors/